services/oc-pmrmrg/src/pmr_mrg/run_model.py

```diff
@@ -0,0 +1,96 @@
+import OpenCOR
+import sys
+import json
+import csv
+
+from pathlib import Path
+
+
+# modelpath = "/home/zhuang/Downloads/guyton_antidiuretic_hormone_2008.cellml"
+inputjson= str(sys.argv[1])
+modelpath = str(sys.argv[2])
+mapfile = str(sys.argv[3])
```

Comment:
```suggestion
```

Reply:
I need this because I create a mapping between the default opencor variables and input_1, input_2... this is because opencor models often have a "\" in the name that the json parser doesn't like.